src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -468,14 +468,15 @@ private void OnStateChanged(NetworkPeerState previous)
         /// Processes an incoming message from the peer and calls subscribed event handlers.
         /// </summary>
         /// <param name="message">Message received from the peer.</param>
```

Comment:
Since this one was renamed from `OnMessageReceived` to `ProcessMessageAsync` I guess it will be better to change this comment too. Smth like `Message to process` maybe.

Reply:
this seems fine to me, the summary claims to process an incoming message from the peer and the param message description says this is the message received from the peer, `message received` seems to have the same meaning as `incoming message`, so i can't see a problem there, makes perfect sense to me